superset/assets/src/explore/components/controls/SubscriberLayer.jsx

```diff
@@ -254,10 +258,16 @@ export default class SubscriberLayer extends React.PureComponent {
         }
       });
 
+      this.state.subscribe_columns.map(column =>
+        column['actions'] = ['APPLY_FILTER']
+      );
+
       subscription['actions'] = [
         "APPLY_FILTER"
       ];
 
+      subscription['useAsModal'] ? subscription['actions'].push('USE_AS_MODAL') : subscription['actions'];
```

Comment:
(question) I do not understand the nature of this inline if since their is not value attribution.
Couldn't this be rewritten as follow:
```javascript
if (subscription['useAsModal']) {
   subscription['actions'].push('USE_AS_MODAL')
}
```

Reply:
will update